### ISSUE TITLE
[Project 2] return되는 id

### ISSUE THREAD

**Issue by juimdpp** [swsnu/osspr2021#68]
안녕하세요! 
과제를 수행하는 도중 헷갈리는 부분이 있어 여쭙습니다!
rotation_lock을 불렀을 때 반환되는 lock을 잡은 process순이어야하는 조건이 추가로 붙나요? 아니면 unique하기만 하면 되는건가요 (순차적이지도 않아도 됨)? 
예를 들어서:
process A: lock request --------------------------> return 1;
process B: ------------- lock request ---> return 0;
처럼 lock request가 실제로 들어온건 A가 먼저인데 실제로 scheduler가 B가 lock을 먼저 잡게 했다고 하면 return 값이 A에 대해서 0이어야할지, B에 대해서 0이어야할지 헷갈립니다.

알려주시면 감사하겠습니다!

**Comment by jaywonchung** [comment]
Nope, uniqueness suffices. In your situation, `A` can even receive lock ID `1000` and `B` `3`, as long as `3` and `1000` are unique non-negative numbers across calls to `rotation_lock`.

**Comment by juimdpp** [comment]
아하 빠른 답변 감사합니다!
